File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.java
package ar.com.wolox.android.example.ui.home;

import android.content.Intent;

import ar.com.wolox.android.R;
import ar.com.wolox.wolmo.core.activity.WolmoActivity;

public class HomeActivity extends WolmoActivity {

    @Override
    protected int layout() {
        return R.layout.activity_home;

    }

    @Override
    protected void init() {
        replaceFragment(R.id.vActivityHome, new HomeFragment());
    }
}

File: app/src/main/java/ar/com/wolox/android/examples/ui/home/HomeActivity.kt
package ar.com.wolox.android.examples.ui.home

import ar.com.wolox.android.examples.R
import ar.com.wolox.wolmo.core.activity.WolmoActivity

class HomeActivity : WolmoActivity() {

    override fun layout(): Int {
        return R.layout.activity_home
    }

    override fun init() {
        replaceFragment(R.id.vActivityHome, HomeFragment())
    }
}